README.md

```diff
@@ -1,15 +1,23 @@
 # TDR AWS Accounts
 
-This repository contains code to configure AWS accounts to support the TDR infrastructure and application
+This repository contains code to configure AWS accounts to support the TDR infrastructure and application.
+There are two scripting languages used:
+* Terraform
+* Python
+
+Code using each language is deployed separately, see sections below.
```

Comment:
Are you intending the code in this repo to be run in Jenkins, and so the manual instructions below (which are really clear, thanks!) are a kind of emergency backup? I think it would be helpful to add a line explaining.

Reply:
Thanks, the instructions are for accounts not managed by Jenkins, I'll add this information to the README once the Jenkins deployment is working.